docker/build.Dockerfile

```diff
@@ -87,5 +87,12 @@ RUN  CXX_COMPILER="g++" \
     && cmake --build . --target check \
     && cmake --build . --target install
 
-# Test Vulkan SDK
-RUN xvfb-run vkcube --c 1000 && echo Done!
+# Enable perfomance layers and test with `vkcube`
+RUN export LD_LIBRARY_PATH=/performance-layers/build:$LD_LIBRARY_PATH \ 
+    && export VK_INSTANCE_LAYERS=VK_LAYER_STADIA_pipeline_compile_time \
+    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_runtime \
+    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_cache_sideload \
+    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_memory_usage \
+    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_frame_time \
+    && export VK_LAYER_PATH=/performance-layers/layer \
+    && xvfb-run vkcube --c 1000 && echo Done!
```

Comment:
1000 frames take ~17s to render on my laptop, should we lower it to 100?

Reply:
Or does it take less time with xvfb than with a physical dispaly?

Reply:
It took about 2s on my machine both with physical display and xvfb. I'll bring it down to 100.